### PR TITLE
Add CTA component

### PR DESCRIPTION
**Pull request by callumacrae**
money:
image

uswitch:
image

The Uswitch one doesn't match the design as the fonts, colours, and primary button styling don't match figma - outside of the scope of this PR.

### CODE REVIEW

File: src/elements/cta/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx, Styled, useThemeUI } from 'theme-ui'
+import { ButtonLink } from '@uswitch/trustyle.button-link'
+
+interface CTAProps extends React.OlHTMLAttributes<HTMLUListElement> {
```
Comment (callumacrae) [inline src/elements/cta/src/index.tsx]: not an ul

File: src/themes/uswitch/theme.json [hunk 2]
```diff
@@ -340,6 +340,21 @@
     }
   },
 
+  "cta": {
+    "main": {
+      "borderWidth": 1,
+      "borderStyle": "solid",
+      "borderColor": "light-3",
+      "borderLeftWidth": 10,
+      "borderLeftColor": "#A3BCFF",
```
Comment (jonathan-fielding) [inline src/themes/uswitch/theme.json]: variable

**Comment by jonathan-fielding** [comment]
The CTA isnt very flexible without a border bottom as we wont be able to just place it anywhere

File: src/themes/uswitch/theme.json (hunk 2 above)
Comment (callumacrae) [reply src/themes/uswitch/theme.json]: See PR description - the colours aren't done in the uswitch theme yet. This can be amended as part of that work

**Comment by callumacrae** [comment]
@jonathan-fielding Can components have knowledge of what they are surrounded by? Otherwise it would have to be baked into the content in contentful, which would make the content itself inflexible (couldn't reuse the same block in the middle of a page and the bottom of a page)

**Pull request merged**